webextensions/api/privacy.json

```diff
@@ -478,11 +470,9 @@
             "__compat": {
```

Comment:
@queengooborg `hyperlinkAuditingEnabled` wasn't one of the additions in the referenced PR. The blame links back to Split webext jsons (https://github.com/mdn/browser-compat-data/pull/488) - so it's addition was certainly earlier.

Reply:
(only because you referenced it in another one of the PRs) but it looks like this was added in https://github.com/mdn/browser-compat-data/pull/296 which equates to 59.